template/private-accounts.json

```diff
@@ -1,8 +1,6 @@
 {
-    "comment": "It is highly recommended to .gitignore this file",
-    "privateAccount":
-    {
-      "address": "",
-      "mnemonic": ""
-    }
+	"comment": "It is highly recommended to .gitignore this file",
```

Comment:
ca we directly add it to git ignore?

Reply:
if we add it to .gitignore we will ignore the file itself.

Reply:
the idea is to add this file in template/gitignore so when the user inits a sample project, Cudos blast will copy gitignore and private-accounts folders, so in the new dir private-accounts will be automatically ignored